charts/oes/values.yaml

```diff
@@ -203,6 +213,11 @@ gate:
     ## Reach out over support@opsmx.com for pre-configured user credentials
     fileBasedAuthentication:
       enabled: false
+    oesui:
+      externalUrl: ^https?://(?:localhost|OES_UI_LOADBALANCER_IP|opsmx.com)(?::[1-9]\d*)?/?
```

Comment:
Let's define OES-UI url and OES-Gate url at a single place and use them all over. Not define it under every section